docs/docs/training-data-format.mdx

```diff
@@ -673,7 +673,7 @@ stories:
   - action: utter_ask_price
 ```
 
-Running the tests can be done using the CLI:
+You can run the tests using the following command:
```

Comment:
Please update anywhere in this section that refers to `test conversations` to `test stories`